lib/_kernel.scm

```diff
@@ -4279,6 +4279,12 @@ end-of-code
             scheme-object   ;; device
    "___os_device_stream_open_process"))
 
+(define-prim ##os-device-open-raw
```

Comment:
Do you prefer the ffi code to have to define the primitive on its own?
Ie instead of having the ffi interface in _kernel.scm to have it in user code?
I don't have any problem with removing the function from Scheme code in this case, but see comment above per convenience.